river/tree/hoeffding_tree_regressor.py

```diff
@@ -406,6 +449,24 @@ def _attempt_to_split(self, node: LearningNode, parent: SplitNode, parent_idx: i
                 self._n_inactive_leaves += 1
                 self._n_active_leaves -= 1
             else:
+                # Update the features' variance estimators if QO is used
+                if self.attr_obs == self._QO and self._qo_std_div:
+                    for feat_id, ao in node.attribute_observers.items():
+                        if not ao.is_numeric:
+                            continue
+
+                        # Update the var estimates using the estimators carried by the QO instances
+                        if feat_id in self._feat_var:
+                            self._feat_var[feat_id] += ao.x_var
+                        else:
+                            self._feat_var[feat_id] = ao.x_var
+
+                        self._qo_radii[feat_id] = {
+                            # Update radii estimates
+                            "radius": math.sqrt(self._feat_var[feat_id].get())
+                            / self._qo_std_div
```

Comment:
Is it worth catching a `ZeroDivisionError`?

Reply:
Good catch! I ensured that the denominator value is always greater than zero in commit [6fc5dc5](https://github.com/online-ml/river/pull/476/commits/6fc5dc54f897fe383e62ca78a5cdf33f153891d9).

Reply:
Anyway, this solution is temporary. In the next PR, by passing the splitters directly (yes, I am getting used to this name :-D), we can simplify this portion.